### ISSUE TITLE
Menu bar animation/change

### ISSUE THREAD

**Issue by MiguelJeme** [yescomment/jm-srf#4]
When scrolling down past the header section animating the menu bar.

Morphing the middle ribbon from _full name logo_ into _SRF icon_

<img width="986" alt="Screenshot 2024-03-06 at 11 10 47 AM" src="https://github.com/yescomment/jm-srf/assets/161371893/d1dd62ad-819e-4cc5-bf9b-680e7d20717d">

**Comment by unitof** [comment]
@MiguelJeme do you like the logo moving to left side at narrow layouts?
<img width="436" alt="46928" src="https://github.com/yescomment/jm-srf/assets/2448782/ff08357b-da4d-4a88-8118-660096e5e50b">

The text to left vs. bottom is making it hard to morph but I'll return to this next week.

For now, just making it full.

**Comment by MiguelJeme** [comment]
Yeah the left looks nice though the side scroll doesnt feel intutive. i know you mentioned a challenge with hamburger menu bubt do you think its doable at some future point.

**Comment by unitof** [comment]
I can do a hamburger/expanding menu later on, yeah. Will redo the menu system them. Leaving this open to remind myself.

Want to do kind of a drawer-style slidedown.

**Comment by MiguelJeme** [comment]
**### New menu design**
<img width="945" alt="Screenshot 2024-05-29 at 4 49 16 PM" src="https://github.com/yescomment/jm-srf/assets/161371893/4f141fe0-12c8-4a2a-a028-4325dcb45e03">
